.github/workflows/tlaplus.yml

```diff
@@ -12,48 +12,54 @@ on:
 jobs:
   model-checking:
     name: Model Checking
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, 1ES.Pool=gha-virtual-ccf-sub]
+    container:
+      image: ccfmsrc.azurecr.io/ccf/ci:05-09-2023-virtual-clang15
 
     steps:
       - uses: actions/checkout@v3
-      - run: python ./tla/install_deps.py
+      - run: |
+          sudo apt update
+          sudo apt install -y default-jre
+          python3 ./tla/install_deps.py
 
       - name: MCccfraft.cfg
         run: |
-          set -exo pipefail
           cd tla/
-          ./tlc.sh -workers auto MCccfraft.tla 2>&1 | tee MCccfraft.out
```

Comment:
I think @lemmy added the stdout files, are you still using them?

Reply:
I seem to have erased that now when updating the description, but the default shell GHA uses is sh -> dash, which does not support pipefail. And so if we pipe commands, we potentially drop errors from any but the last command, which means that we can't tell when the validation fails. That's an awkward price to pay for the .out file.